src/web/screens/home.tsx

```diff
@@ -26,21 +26,14 @@ function Home() {
         const sharingList = await GetSharingList();
         const manualSites = sharingList.allowed_sites;
         const allowedTypes = sharingList.allowed_types;
-        const allowedTypeMap = new Map<ClientType, boolean>();
+        const allowedTypeSet = new Set<ClientType>();
```

Comment:
nit: could just be `new Set<ClientType>(allowedTypes);`

Reply:
Good idea.  I changed it.